.pre-commit-hooks.yaml

```diff
@@ -28,3 +28,16 @@
   pass_filenames: false
   additional_dependencies: []
   minimum_pre_commit_version: "2.9.2"
+- id: uv-sync
+  name: uv-sync
+  description: "Automatically run 'uv sync' on your workspace after a checkout or pull"
```

Comment:
"Workspace" has a technical meaning in the land of `uv`, so this description might be a little misleading. If I were running this on a uv workspace I would probably add the additional arg `--all-packages` in my pre-commit config.

Reply:
OK I wrote this as git workspace but I understand the confusion and will reword it.
I'll add a section in the readme to mention the `--all-packages` argument that may be added.
However my python script will not look up to nested files to determine whether sync is required.